pkg/i2gw/providers/ingressnginx/ingressnginx.go

```diff
@@ -37,6 +37,7 @@ type Provider struct {
 	storage        *storage
 	resourceReader *resourceReader
 	converter      *converter
+	extConverter   *extensionConverter
```

Comment:
I just picked on ingress-nginx to leave my comments, but most of them are applicable for all providers. 

These names probably deserves better naming

Reply:
Updated. Now it is called `irToGatewayResourcesConverter`.